cylc/flow/scheduler.py

```diff
@@ -1125,10 +1120,6 @@ def load_flow_file(self, is_reload=False):
             handle.write(b"# cylc-version: %s\n" % CYLC_VERSION.encode())
             printcfg(self.config.cfg, none_str=None, handle=handle)
 
-        if not self.config.initial_point and not self.is_restart:
-            LOG.warning('No initial cycle point provided - no cycling tasks '
-                        'will be loaded.')
-
```

Comment:
Again pretty sure this condition is no longer possible (`config.initial_point` should be set by this point)

Reply:
Legit.